hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/management/DeploymentView.java

```diff
@@ -407,11 +407,15 @@ private void showOrHideFilterButtons(final int browserWidth) {
     protected Map<Class<?>, RefreshableContainer> getSupportedPushEvents() {
         final Map<Class<?>, RefreshableContainer> supportedEvents = Maps.newHashMapWithExpectedSize(10);
 
-        supportedEvents.put(TargetCreatedEventContainer.class, targetTableLayout.getTable());
-        supportedEvents.put(TargetDeletedEventContainer.class, targetTableLayout.getTable());
+        if (targetTableLayout != null) {
```

Comment:
I'd prefer to use same logic as already in place in other methods of this class. What do you think:
```
if (permChecker.hasTargetReadPermission()) {
    supportedEvents.put(TargetCreatedEventContainer.class, targetTableLayout.getTable());
    supportedEvents.put(TargetDeletedEventContainer.class, targetTableLayout.getTable());
}
```

Reply:
I was thinking about performance, since reading the security context is quite slow in comparison to a simple null check. 
But you are right, to keep it consistent within the class I should reuse the "permChecker".
I'll update the PR :)